packages/docusaurus-theme-classic/src/theme/MDXComponents/index.js

```diff
@@ -16,6 +16,9 @@ export default {
   code: (props) => {
     const {children} = props;
     if (typeof children === 'string') {
+      if (children.indexOf('\n') === -1) {
```

Comment:
This makes sense. Suggestion:

```js
if (!children.includes('\n')) {
  ...
}
```

Reply:
I did this because `indexOf` is the fastest of them.

Reply:
@yangshun so should I replace `indexOf` with `includes`? Or can I leave first one?

Reply:
Yeah feel free. I think the perf difference here is negligible. I'd optimize for readability. But up to you.

Reply:
:heavy_check_mark: